.github/workflows/move-to-in-review.yml

```diff
@@ -0,0 +1,26 @@
+name: move-card-to-in-review
+
+on:
+  pull_request:
+    types: [opened, reopened, ready_for_review, synchronize, edited]
+
+jobs:
+  extract-issue-numbers:
+    runs-on: ubuntu-latest
+    outputs:
+      issues: ${{ steps.extract.outputs.issues }}
+    steps:
+      - id: extract
+        uses: MatrixFrog/pr-extract-issues@2684b0f1d7b1671b7fb9ab8bfbb9b2b88983e6c5
```

Comment:
this is https://github.com/marketplace/actions/pr-extract-issues except I forked it to fix a few small issues. My fork is here: https://github.com/MatrixFrog/pr-extract-issues